miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/WebViewDialog.kt

```diff
@@ -36,6 +38,17 @@ internal fun onShowDialog(
     return true
 }
 
+internal fun openNonSSLDialog(context: Context, url: String) =
+    AlertDialog.Builder(context)
+        .setMessage("This link is unsafe. If you would like to proceed, it will be opened in your native browser.")
```

Comment:
Is it possible to show the full `url` as well? Here is my idea. We can do other UI improvement if we want. 

![Screen Shot 2020-10-21 at 17 34 15](https://user-images.githubusercontent.com/66930373/96694717-c0417d80-13c3-11eb-95a2-34bf107bd706.png)

Reply:
Some url are very long so I think only message is fine enough.

Reply:
Ok, we can take care of the UI in another time.